data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/deserializers/UsageRuleDeserializer.java

```diff
@@ -77,6 +77,8 @@ private static FreeToUse deserializeFreeToUseUsageRule(JsonParser jsonParser, Si
         while (!jsonParser.nextToken().isStructEnd()) {
 
             switch (jsonParser.getCurrentName()) {
+                case TYPE:
+                    break;
```

Comment:
at least add sonParser.nextToken(), otherwise you will get a NullPointerException at the next `switch (jsonParser.getCurrentName())`
but i can also be removed :rocket: